### PR TITLE
Update dto API completed

### PR DESCRIPTION
**Pull request by Jittojoyes98**
Resolves https://github.com/ToolJet/ToolJet/issues/5400

### CODE REVIEW

File: server/src/dto/organization.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { Transform } from 'class-transformer';
```
Comment (gsmithun4) [inline server/src/dto/organization.ts:1]: Shall we rename this file to `organization.dto.ts`

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

File: server/src/dto/organization.ts (hunk 1 above)
Comment (Jittojoyes98) [reply server/src/dto/organization.ts:1]: yeah ok, will fix

File: server/src/dto/organization.dto.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+export class OrganizationCreateDto {
+  @IsString()
+  @Transform(({ value }) => sanitizeInput(value))
+  @IsNotEmpty()
+  @MaxLength(25, { message: 'Name cannot be longer than 25 characters' })
+  name: string;
+}
+
+export class OrganizationUpdateDto {
+  @IsString()
```
Comment (gsmithun4) [inline server/src/dto/organization.dto.ts:15]: shall we change the order?
```
  @IsOptional()
  @IsString()
```
for all fields

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

**Review by gsmithun4 — APPROVED** [review]

**Comment by gsmithun4** [comment]
Hi @Jittojoyes98 , the messages added is not showing in front end, please get the message from error object and set the toast
https://github.com/ToolJet/ToolJet/blob/improvement/workspace-id-scope-be/frontend/src/ManageSSO/GeneralSettings.jsx#L34
```
(err) => {
        setSaving(false);
        toast.error(err?.data?.message || 'Error while saving SSO configurations', {
          position: 'top-center',
        });
      }
```

**Comment by gsmithun4** [comment]
Same here: https://github.com/ToolJet/ToolJet/blob/develop/frontend/src/_components/OrganizationManager/EditOrganization.jsx#L25

**Review by gsmithun4 — CHANGES_REQUESTED** [review]
display error messages

**Comment by Jittojoyes98** [comment]
In the second toast message should i need to place somewhere, like the other toast which has the position on the  top.

**Review by gsmithun4 — APPROVED** [review]

**Pull request merged**